CHANGELOG.md

```diff
@@ -1,13 +1,14 @@
 #### Changes
-* Python: Python: FT.PROFILE command added ([#2543](https://github.com/valkey-io/valkey-glide/pull/2543))
-* Python: Python: FT.AGGREGATE command added([#2530](https://github.com/valkey-io/valkey-glide/pull/2530))
+* Python: FT.PROFILE command added ([#2543](https://github.com/valkey-io/valkey-glide/pull/2543))
+* Python: FT.AGGREGATE command added ([#2530](https://github.com/valkey-io/valkey-glide/pull/2530))
 * Python: Add JSON.OBJLEN command ([#2495](https://github.com/valkey-io/valkey-glide/pull/2495))
-* Python: FT.EXPLAIN and FT.EXPLAINCLI commands added([#2508](https://github.com/valkey-io/valkey-glide/pull/2508))
-* Python: Python FT.INFO command added([#2429](https://github.com/valkey-io/valkey-glide/pull/2494))
-* Python: Add FT.SEARCH command([#2470](https://github.com/valkey-io/valkey-glide/pull/2470))
-* Python: Add commands FT.ALIASADD, FT.ALIASDEL, FT.ALIASUPDATE([#2471](https://github.com/valkey-io/valkey-glide/pull/2471))
-* Python: Python FT.DROPINDEX command ([#2437](https://github.com/valkey-io/valkey-glide/pull/2437))
-* Python: Python: Added FT.CREATE command([#2413](https://github.com/valkey-io/valkey-glide/pull/2413))
+* Python: FT.EXPLAIN and FT.EXPLAINCLI commands added ([#2508](https://github.com/valkey-io/valkey-glide/pull/2508))
+* Python: FT.INFO command added ([#2429](https://github.com/valkey-io/valkey-glide/pull/2494))
+* Python: Add FT.SEARCH command ([#2470](https://github.com/valkey-io/valkey-glide/pull/2470))
+* Python: Add commands FT.ALIASADD, FT.ALIASDEL, FT.ALIASUPDATE ([#2471](https://github.com/valkey-io/valkey-glide/pull/2471))
+* Python: Add FT.DROPINDEX command ([#2437](https://github.com/valkey-io/valkey-glide/pull/2437))
+* Python: Added FT.CREATE command ([#2413](https://github.com/valkey-io/valkey-glide/pull/2413))
+* Python: Added `FT._LIST` command ([#2571](https://github.com/valkey-io/valkey-glide/pull/2571))
```

Comment:
We shouldn't be inconsistent with how we format the command names.

Reply:
I like MD more, personally, but I don't have a strong opinion either way.

Reply:
Will update changelog in another PR then